documentation/src/main/asciidoc/topics/proc_server_testing_datasources.adoc

```diff
@@ -0,0 +1,27 @@
+[id='testing_datasources']
+= Testing Data Sources
+Data Sources can be tested via the CLI to verify that the connection to the database is functioning correctly.
+
+.Procedure
+
+. Start the CLI.
++
+[source,options="nowrap",subs=attributes+]
+----
+$ bin/cli.sh
+[disconnected]>
+----
++
+. List all data sources:
++
+[source,options="nowrap",subs=attributes+]
+----
+[//containers/default]> server datasource ls
+----
++
+. Test a data source connection.
++
+[source,options="nowrap",subs=attributes+]
+----
+[//containers/default]> server datasource test my-datasource
+----
```

Comment:
Could we provide a hint for resolving connection issues? 

I can open a separate docs enhancement for this if you want to get this merged sooner. But I feel like we could expand this out with some basic troubleshooting.

Reply:
although maybe those troubleshooting hints could go into logging descriptions.

Reply:
I've added more details to the failure message

Reply:
nice!